src/components/text-editor/__internal__/toolbar/toolbar-button/toolbar-button.d.ts

```diff
@@ -6,9 +6,6 @@ export interface ToolbarButtonProps {
   activated: boolean;
   onKeyDown: (event: React.KeyboardEvent<HTMLElement>) => void;
   onMouseDown: (event: React.MouseEvent<HTMLElement>) => void;
-  onMouseOver: (event: React.MouseEvent<HTMLElement>) => void;
-  onMouseLeave: (event: React.MouseEvent<HTMLElement>) => void;
-  tooltipVisible?: boolean;
```

Comment:
I think we want to keep `tooltipVisible` in here, and add `tooltipMessage` as well

Reply:
Yeah good point I forgot to add them back in when I refactored it earlier

Reply:
The tooltip functionality is now handled in the `Toolbar`, I've added the event handler callbacks in though